charts/ingress-nginx/values.yaml

```diff
@@ -768,22 +768,24 @@ revisionHistoryLimit: 10
 ##
 defaultBackend:
   ##
-  enabled: false
+  enabled: true
 
   name: defaultbackend
   image:
-    registry: registry.k8s.io
-    image: defaultbackend-amd64
+    repository: nginx
+    tag: alpine
```

Comment:
isn't this tag a bit too open?

Reply:
nginx:1.23.1-alpine For the moment, in the future we will create the backend with Go